src/main/java/io/strimzi/kafka/bridge/tracing/OpenTelemetryHandle.java

```diff
@@ -65,11 +68,26 @@ public String serviceName(BridgeConfig config) {
     @Override
     public void initialize() {
         System.setProperty("otel.metrics.exporter", "none"); // disable metrics
+        final String configuredOtlpProtocol = resolveConfiguredOtelProtocol();
+        if (configuredOtlpProtocol == null || configuredOtlpProtocol.equalsIgnoreCase("grpc")) {
+            LOGGER.warn("The HTTP-Kafka bridge is deprecating OTLP GRPC support in favour of plain HTTP transports. Please switch to http/protobuf and update collector addresses to match");
+        }
```

Comment:
I was wondering if we should apply a default of `http/protobuf` here if nothing is set, but given we have to be configured with the appropriate port that sounds like a dubious choice.